src/modules/70-pipeline/components/ArtifactsSelection/ArtifactInterface.ts

```diff
@@ -87,8 +87,38 @@ export interface ImagePathTypes {
   repositoryFormat?: string
 }
 
-export interface CustomArtifactSource extends ImagePathTypes {
-  version: string
+export interface variableInterface {
```

Comment:
```suggestion
export interface VariableInterface {
```
Interface must start with uppercase